sdl-core/src/main/scala/io/smartdatalake/workflow/action/generic/transformer/EncryptColumnsTransformer.scala

```diff
@@ -37,16 +37,16 @@ import org.apache.spark.sql.functions.{col, udf}
 trait EncryptDecrypt {
   def keyAsBytes: Array[Byte]
 
-  val cryptUDF: UserDefinedFunction = udf(encrypt _)
-  private val ALGORITHM_STRING: String = "AES/GCM/PKCS5Padding"
+  val cryptUDF: UserDefinedFunction = udf(encryptGCM _)
```

Comment:
Why is this statically initialized to GCM?

Reply:
that was meant to be overriden, but with an default value, but now its handled differently. 👍